projects/plugins/jetpack/extensions/blocks/contact-form/transforms.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * External dependencies
+ */
+import { pickBy, identity } from 'lodash';
+import { createBlock } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+
+const getAttributeValue = ( tag, att, content ) => {
+	const dbqts = content.match( new RegExp( `\\[${tag}[^\\]]* ${att}="([^"]*)"`, 'im' ) );
```

Comment:
I would just spell these const names out in full `doubleQuotes` ... just a minor thing, but saves a few seconds of overhead working out what these abbreviations mean